modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/OperationMgtService.java

```diff
@@ -46,26 +46,41 @@
 
 	private static Log log = LogFactory.getLog(OperationMgtService.class);
 
-	@GET
+	@PUT
 	@Produces(MediaType.APPLICATION_JSON)
 	@Path("{id}")
 	public List<? extends org.wso2.carbon.device.mgt.common.operation.mgt.Operation> getPendingOperations
-			(@HeaderParam("Accept") String acceptHeader, @PathParam("id") String id) {
+			(@HeaderParam("Accept") String acceptHeader, @PathParam("id") String id,
```

Comment:
while the response is built according to the content-type specified by the accept-header, can we just limit the response to JSON by specifying "@Produces(MediaType.APPLICATION_JSON)"?